services/ui-auth/serverless.yml

```diff
@@ -77,6 +84,52 @@ resources:
         ExplicitAuthFlows:
           - ADMIN_NO_SRP_AUTH
         GenerateSecret: false
+        AllowedOAuthFlows:
+          - implicit
+        AllowedOAuthFlowsUserPoolClient: true
+        AllowedOAuthScopes:
+          - email
+          - openid
+          - aws.cognito.signin.user.admin # Needed to call Auth.currentUserInfo()
+        CallbackURLs:
+          - ${self:custom.application_endpoint_url}
+          - http://localhost:3000/
+        DefaultRedirectURI: ${self:custom.application_endpoint_url}
+        LogoutURLs:
+          - ${self:custom.application_endpoint_url}
+          - http://localhost:3000/
+        SupportedIdentityProviders:
+          - COGNITO
```

Comment:
This is where, if desired, modification could be made to make COGNITO support conditional.  For quickstart purposes, we will always support direct cogntio signup/login